modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyAuditBarChart.js

```diff
@@ -26,24 +24,29 @@ import {
 
 import {BAR_CHART} from '../utils/constants';
 
+const LEARN_HOW_URL =
+	'https://help.liferay.com/hc/en-us/articles/360028820492-Defining-Categories-for-Content';
+
 export default function EmptyAuditBarChart() {
 	return (
 		<>
 			<ClayEmptyState
 				className="empty-state text-center"
-				description={Liferay.Language.get(
-					'create-marketing-categories-to-label-and-audit-your-content'
-				)}
-				title={Liferay.Language.get('there-is-no-data')}
-			>
-				<ClayButton displayType="secondary">
-					{Liferay.Language.get('add-marketing-categories')}
-					<ClayIcon
-						className="inline-item inline-item-after"
-						symbol="shortcut"
+				description={
+					<div
+						dangerouslySetInnerHTML={{
+							__html: Liferay.Util.sub(
+								Liferay.Language.get(
+									'learn-how-to-tailor-categories-to-your-needs'
+								),
+								`<a href=${LEARN_HOW_URL} target="_blank">`,
+								'</a>'
+							),
+						}}
```

Comment:
For reference, [Slack thread on this](https://liferay.slack.com/archives/CNBG06JS3/p1594204672332000).

Was suggested that you could do this without `dangerouslySetInnerHTML` ([codesandbox](https://codesandbox.io/s/broken-violet-w3yb2?file=/src/App.js)), so if you're sure that's not going to work, I'm fine with this; there is [at least one other example that does it the same way you're doing it](https://github.com/liferay/liferay-portal/blob/dfa080768228be927b37085a56e541c05ce0f072/modules/apps/redirect/redirect-web/src/main/resources/META-INF/resources/js/DestinationUrlInput.js#L112-L118) already in liferay-portal.

Reply:
I tried @bryceosterhaus suggestion, but the portal was rendering the HTML as a plain text 😢